docs/news/index.qmd

```diff
@@ -9,6 +9,90 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2023.03.0
+
+**"Cherry Blossom"**
+
+>Date: 2023-03-16
+
+### New
+
+#### RStudio IDE
+- Adopt panmirror (visual editor) migration to quarto-dev/quarto project (#12316)
```

Comment:
```suggestion
- Adopt pan mirror (visual editor) migration to quarto-dev/quarto project (#12316)
```
Is panmirror one or two words? Unless this is something specific, I think that it is one word?